### PR TITLE
reformat dataset

### PR DESCRIPTION
**Pull request by chenyuqi990215**

### CODE REVIEW

File: physiopro/dataset/event.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import numpy as np
```
Comment (suiguoxin) [inline physiopro/dataset/event.py:1]: add MS header to all the .py files

File: physiopro/dataset/__init__.py [hunk 2]
```diff
@@ -1,4 +1,10 @@
...
 from .ts import TSDataset
-from .dataset_wrapper import DATASETWRAPPER, MPWrapper
-from .tusz import TUSZForecastDataset
+from .event import EventDataset
+from .seed import SEEDDataset
+from .detection import DetectionDataset
+
+from .preprocessor import Preprocessor
+from .transformation import *
+from .dataset_wrapper import DATASETWRAPPER, MPWrapper
```
Comment (suiguoxin) [inline physiopro/dataset/__init__.py:10]: Please fix obvious pylint issues

File: physiopro/dataset/dataset_wrapper.py [hunk 3]
```diff
@@ -1,5 +1,3 @@
-# Copyright (c) Microsoft Corporation.
-# Licensed under the MIT License.
```
Comment (suiguoxin) [inline physiopro/dataset/dataset_wrapper.py:2]: why remove the header?

File: physiopro/dataset/seed.py [hunk 4]
```diff
@@ -0,0 +1,231 @@
+from typing import Optional, Any
```
Comment (suiguoxin) [inline physiopro/dataset/seed.py:1]: Shall we add this part in a dedicated PR?

**Review by suiguoxin — COMMENTED** [review]
Please add the schema description of preprocessor

File: physiopro/dataset/event.py (hunk 1 above)
Comment (chenyuqi990215) [reply physiopro/dataset/event.py]: thanks for reminding!

File: physiopro/dataset/dataset_wrapper.py (hunk 3 above)
Comment (chenyuqi990215) [reply physiopro/dataset/dataset_wrapper.py]: fixed.

File: physiopro/dataset/seed.py (hunk 4 above)
Comment (chenyuqi990215) [reply physiopro/dataset/seed.py]: sure.

File: physiopro/dataset/__init__.py (hunk 2 above)
Comment (chenyuqi990215) [reply physiopro/dataset/__init__.py:8]: what do you mean by pyline issues?

Comment (suiguoxin) [reply physiopro/dataset/__init__.py:8]: > what do you mean by pyline issues?

Try to run pylint over the files you have touched, e.g., `pylint physiopro/dataset`, `pylint physiopro/transformation`, etc.

**Review by suiguoxin — APPROVED** [review]

**Pull request merged**